pkg/operation/seed/components.go

```diff
@@ -136,6 +145,111 @@ func defaultGardenerResourceManager(c client.Client, seedClientVersion string, i
 	return gardenerResourceManager, nil
 }
 
+func defaultIstio(ctx context.Context,
+	seedClient client.Client,
+	imageVector imagevector.ImageVector,
+	chartApplier kubernetes.ChartApplier,
+	chartRenderer chartrenderer.Interface,
+	seed *Seed,
+	conf *config.GardenletConfiguration,
+	sniEnabledOrInUse bool) ([]component.DeployWaiter, error) {
+	istiodImage, err := imageVector.FindImage(images.ImageNameIstioIstiod)
+	if err != nil {
+		return nil, err
+	}
+
+	igwImage, err := imageVector.FindImage(images.ImageNameIstioProxy)
+	if err != nil {
+		return nil, err
+	}
+
+	istioCRDs := istio.NewIstioCRD(chartApplier, charts.Path, seedClient)
```

Comment:
Let's rather move this to https://github.com/gardener/gardener/blob/f08eff4f443dd466659dbe7a4e2271d9d4980640/pkg/operation/seed/seed.go#L383-L391 where the other CRDs are being created.